### ISSUE TITLE
Wrong display in wallabag (uxdesign.cc)

### ISSUE THREAD

**Issue by papernoise** [wallabag/wallabag#6050]
- [x] I've tried myself without success
- [x] I've replaced `HOST` in the issue title with the host of the URL that can't be fetched (ie: `nytimes.com`, `20minutes.fr`, `bbc.com`, etc.)

**Content related:**
 - URL: [[full url of the content]](https://uxdesign.cc/do-your-icons-mean-what-you-think-they-mean-fa6f255d56ab) (but all articles from the domain produce the same error)
 - wallabag version: 2.5.2 (app.wallabag.it)

**Describe what's wrong:**
- Wallbag cannot parse the articles on that domain.
- Wallabag displays "wallabag can't retrieve contents for this article" instead of the article content.
- f43 gives this error: "Circular redirection detected" and also fails to fetch the content. At the bottom of the page it then states: "We failed to make this item readable, the default text from the feed item will be displayed instead."

**further notes:**
- uxdesign.cc is just a custom domain for a medium.com account. Articles with the regular medium.com URL parse correctly.
- the issue appeared recently, about 1-2 weeks ago.

**Comment by Kdecherf** [comment]
I can't reproduce it, does this issue still persist?

**Comment by papernoise** [comment]
Hi! Thanks for checking this! It does indeed still persist. Occasionally it works, but most of the time Wallabag seems unable to parse anything coming from medium.com. Unfortunately a lot of articles are hosted on that platform.

Here's a few I've tried and which don't work:

https://uxdesign.cc/how-i-used-midjourney-to-design-a-brand-identity-394cf9ddaeed

https://blog.angular.io/angular-v16-is-here-4d7a28ec680d

https://entrepreneurshandbook.co/the-bang-with-friends-story-under-the-covers-77f757bdb822

I've tried to add them both using the Wallabagger Firefox plug-in and manually on wallabag.it
The result is always the same, Wallabag says "wallabag can't retrieve contents for this article."
